<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickDialog</class>
 <widget class="QDialog" name="ColorPickDialog">
  <property name="windowTitle">
   <string>Color</string>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed"/>
  </property>
  <layout class="QGridLayout" name="dialogLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="colorLayout"/>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
